JPS_BASE_LIB/src/test/java/uk/ac/cam/cares/jps/base/query/StoreRouterTest.java

```diff
@@ -13,158 +16,161 @@
 		Mockito.doReturn(mockStore).when(spyStoreRouter).getRouterStoreClient();
-		
-		//Not in cache
+
+		// Not in cache
 		assertEquals(expected, spyStoreRouter.get(TEST_LABEL));
 		Mockito.verify(spyStoreRouter, Mockito.times(1)).getRouterStoreClient();
 		Mockito.verify(spyStoreRouter, Mockito.times(1)).getFromStore(TEST_LABEL, mockStore);
-		
-		//In cache so getFromStore and getRouterStoreClient are not called again
+
+		// In cache so getFromStore and getRouterStoreClient are not called again
 		assertEquals(expected, spyStoreRouter.get(TEST_LABEL));
 		Mockito.verify(spyStoreRouter, Mockito.times(1)).getRouterStoreClient();
 		Mockito.verify(spyStoreRouter, Mockito.times(1)).getFromStore(TEST_LABEL, mockStore);
 
-		//Label does not exist
+		// Label does not exist
 		String label = "labeldoesnotexist";
 		assertNull(spyStoreRouter.get(label));
 		Mockito.verify(spyStoreRouter, Mockito.times(2)).getRouterStoreClient();
-		Mockito.verify(spyStoreRouter, Mockito.times(1)).getFromStore(label, mockStore); //only called once with these arguments
+		// only called once with these arguments
+		Mockito.verify(spyStoreRouter, Mockito.times(1)).getFromStore(label, mockStore);
 	}
-	
+
 	@Test
-	void testIsFileBasedTargetResourceID() {		
-		//Test different extensions
+	void testIsFileBasedTargetResourceID() {
+		// Test different extensions
 		assertTrue(StoreRouter.isFileBasedTargetResourceID("kb/sgp/singapore/SGTemperatureSensor-001.owl"));
 		assertTrue(StoreRouter.isFileBasedTargetResourceID("kb/sgp/singapore/SGTemperatureSensor-001.rdf"));
 		assertTrue(StoreRouter.isFileBasedTargetResourceID("kb/sgp/singapore/SGTemperatureSensor-001.nt"));
-		
-		//Test IRI
-		assertTrue(StoreRouter.isFileBasedTargetResourceID("http://theworldavatar.com/kb/sgp/singapore/SGTemperatureSensor-001.owl"));
-		
-		//Test not file based resources
+
+		// Test IRI
+		assertTrue(StoreRouter
+				.isFileBasedTargetResourceID("http://theworldavatar.com/kb/sgp/singapore/SGTemperatureSensor-001.owl"));
+
+		// Test not file based resources
 		assertFalse(StoreRouter.isFileBasedTargetResourceID("http://theworldavatar.com/kb/ontokin"));
 		assertFalse(StoreRouter.isFileBasedTargetResourceID("ontokin"));
-		
-		//Invalid file paths
-		assertFalse(StoreRouter.isFileBasedTargetResourceID("kb/sgp:singapore/SGTemperatureSensor-001.owl"));
-		assertFalse(StoreRouter.isFileBasedTargetResourceID("kb/sgp?singapore/SGTemperatureSensor-001.owl"));
-		assertFalse(StoreRouter.isFileBasedTargetResourceID("kb/sgp*singapore/SGTemperatureSensor-001.owl"));
+
+		// Invalid file paths
+		// These are legal file paths in Linux
+		// assertFalse(StoreRouter.isFileBasedTargetResourceID("kb/sgp:singapore/SGTemperatureSensor-001.owl"));
+		// assertFalse(StoreRouter.isFileBasedTargetResourceID("kb/sgp?singapore/SGTemperatureSensor-001.owl"));
+		// assertFalse(StoreRouter.isFileBasedTargetResourceID("kb/sgp*singapore/SGTemperatureSensor-001.owl"));
 	}
```

Comment:
Should these just be deleted?

Reply:
I kept them as a sort of documentation of the method being tested. I considered just changing the expected value based on the operating system but decided that that might not be reliable.